Extensions/dnSpy.BamlDecompiler/Rewrite/MarkupExtensionRewritePass.cs

```diff
@@ -53,6 +54,8 @@ sealed class MarkupExtensionRewritePass : IRewritePass {
 			return doWork;
 		}
 
+		static bool NonInlineAttibute(XAttribute attr) => attr.Name.Namespace == XamlContext.KnownNamespace_Xaml;
```

Comment:
Settling on `IsXamlNsAttr`, because names of reflection member entities that aren't classes derived from `System.Attribute` now appear to be confusing to some of the latest static analysis tools. VS intellisense seems over-eager to strip off `-Attribute` as an aliasing suffx.